.dockerignore

```diff
@@ -0,0 +1,5 @@
+.env
+.git
+.venv
+Dockerfile
+.vscode
```

Comment:
Just a couple of extra files we might consider here based on what I notice around my codebase:
* `**/__pycache__` (note the `**/` is necessary in `.dockerignore` to say to ignore this regardless of whether it's a file/folder at the top level, or whether it's in a sub-folder
* `**/.pytest_cache`